docs/2.1. APIs - Common Keys.md

```diff
@@ -83,20 +84,21 @@ See [Use of URNs](#use-of-urns) below for additional requirements regarding thei
 All Uniform Resource Names (URNs) specified within the scope of the NMOS specifications MUST use the namespace 'urn:x-nmos:'. Manufacturers MAY use their own namespaces to indicate values of URN which are not currently defined within the NMOS namespace.
 
 Where a URN begins 'urn:x-nmos:' it MUST use the following construction:
-*   The URN 'base' is a sequence of one or more names delimited by ':' characters, including the namespace
-*   The URN MAY include one or more 'subclassifications'
-*   The URN MAY include a version number
+
+- The URN 'base' is a sequence of one or more names delimited by ':' characters, including the namespace
+- The URN MAY include one or more 'subclassifications'
+- The URN MAY include a version number
 
 Subclassifications are defined as the portion of the URN which follows the first occurrence of a '.', but prior to any '/' character. By convention, URNs which begin 'urn:x-nmos:' will never include a further ':' after the first occurrence of a '.'.
 
 For example, the URN base 'urn:x-nmos:transport:rtp' above has two subclassifications of 'ucast' and 'mcast'.
 
 API clients interpreting URNs MUST correctly recognise any as-yet undefined subclassifications as being associated with the URN base which appears prior to the first '.' character.
 
-For example, if filtering by the format 'urn:x-nmos:format:video', the filter is required to include any as-yet undefined subclassifications such as 'urn:x-nmos:format:video.raw'.
+For example, if filtering by the format 'urn:x-nmos:format:video', the filter includes any as-yet undefined subclassifications such as 'urn:x-nmos:format:video.raw'.
```

Comment:
Just noting that in the long time since this was originally written, practice has shown that we don't need to add such subclassifications, we use other mechanisms, such as IANA `media_type` in this example, so this might be something we want to change in any v1.4 or v2.0.

Reply:
Made #161 to discuss.